### PR TITLE
Update manage-billing-profiles.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Microsoft approved resellers and Partners to the list as they have access to Billing Page as well(Tested)

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue# https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2738

File: microsoft-365/commerce/billing-and-payments/manage-billing-profiles.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ keywords: billing profile, invoices, charges, managed charges
 
 ::: moniker-end
 
-For commercial customers who buy products and services from Microsoft, billing profiles let you customize what items are included on your invoice, and how you pay your invoices.
+For commercial customers who buy products and services from Microsoft, Microsoft approved resllers or Microsoft Partners, billing profiles let you customize what items are included on your invoice, and how you pay your invoices.
```
Comment (JohanFreelancer9) [inline microsoft-365/commerce/billing-and-payments/manage-billing-profiles.md]:
```suggestion
For commercial customers who buy products and services from Microsoft, Microsoft approved resellers, and Microsoft partners, billing profiles let you customize what items are included on your invoice, and how you pay your invoices.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@cmcatee-MSFT  Gentle reminder, Please merge this PR. Thanks.

**Comment by cmcatee-MSFT** [comment]
Article is undergoing a rewrite, making this PR unnecessary.

**Pull request closed without merging**